test/Feature/AndOrParamsSpec.hs

```diff
@@ -80,6 +80,19 @@ spec =
               {"text_search_vector": "'amus':5 'fair':7 'impossibl':9 'peu':4" },
               {"text_search_vector": "'art':4 'spass':5 'unmog':7"}
             ]|] { matchHeaders = [matchContentTypeJson] }
+
+        when (actualPgVersion >= pgVersion112) $
+          it "can handle fts" $
```

Comment:
Above this test there's a `it "can handle fts"` test, that makes this one look like a duplicate.   

Maybe rename this test as `it "can handle wfts(websearch_to_tsquery)"`

Reply:
Well spotted, thanks.